### DISCUSSION TITLE
Please advise me, First time user. Controling case fans without touching others.

### DISCUSSION THREAD

**Discussion by cursedwraith** [Rem0o/FanControl.Releases#2870] [General]
Hi there,

First of all appreciate the developers hard work for this amazing piece of software.

I am trying to control 6 Lian li fan that i have in my case but i find the program kind of "complicated".
I have played around with it and was able to understand and set some settings however,

- I want to control ONLY the 6 lian li fan that i have based on cpu temp.
- I dont want the Fan control to play with Gpu fan, Cpu fan or my AIO pump. I want those to come from bios instead.
- I noticed that it starts analyzing everything then i was concerned that its actually doings something with gpu and cpu fan  pump so i uninstalled it.

Now that im looking to reinstall, Is it possible to just control the 6 case fan that i have without even adjusting or activating others ? How would i go about that ? what should i check to make sure nothing is being done with cpu gpu etc.

Any advice appreciated. Thanks :)

**Comment by Rem0o** [comment]
Hi,

When you first start the program, you can choose your "sources".   Keep only "motherboard" and "cpu" (to get temperatures) and that's it.  You can change those at any times in the settings The program will then still analyze all sensors available to it, however once it is done, it will reset them to default until you toggle them on. 

TLDR: If a control card is left toggled off, Fancontrol doesn't touch/mess with it.
